e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -683,6 +683,45 @@ describe("scenarios > dashboard", () => {
       cy.findByText("There was a problem displaying this chart.");
     });
   });
+
+  it("should now overflow markdown content in description (metabase#31326)", () => {
+    cy.intercept("GET", "/api/dashboard/1").as("getDashboard");
+    cy.intercept("PUT", "/api/dashboard/1").as("updateDashboard");
+    visitDashboard(1);
+    cy.wait("@getDashboard");
+
+    cy.get("main header").within(() => {
+      cy.icon("info").click();
+    });
+
+    const testMarkdownContent =
+      "{selectall}# Heading 1{enter}{enter}**bold** [link](https://metabase.com){enter}{enter}![alt](https://upload.wikimedia.org/wikipedia/commons/a/a2/Cat_outside.jpg){enter}{enter}This is my description. ";
```

Comment:
If the link to this image dies, it will affect this test.

Can we make this test independent of 3rd parties?

Reply:
We can choose some static images from `resources/frontend_client/app/assets/img/` e.g. `resources/frontend_client/app/assets/img/welcome-modal-2.png` which is 
![image](https://github.com/metabase/metabase/assets/1937582/d40b6e1f-3604-4ef0-be40-f3ea6b744589)